website/public/locales/sv/leaderboard.json.bak

```diff
@@ -0,0 +1,36 @@
+{
+  "accepted": "↪ Accepterad",
```

Comment:
I guess this `.bak` file was accidentally included and should not be part of this PR.